repo/blob/sftp/sftp_storage.go

```diff
@@ -101,6 +101,22 @@ func (s *sftpImpl) IsConnectionClosedError(err error) bool {
 	return false
 }
 
+func (s *sftpStorage) GetCapacity(ctx context.Context) (blob.Capacity, error) {
+	i, err := s.Impl.(*sftpImpl).rec.UsingConnection(ctx, "GetCapacity", func(conn connection.Connection) (interface{}, error) {
+		stat, err := sftpClientFromConnection(conn).StatVFS("/")
```

Comment:
same thing here, should we be passing the actual path and not the root?

Reply:
@jkowalski (as a drive by reviewer) that makes some logic - since we'd want the free space at that path ( I suspect even bucket based backends you may, as remote quota policies may be set?).

The paranoid monkey in me has seen ops being quickly create symlinks to other file systems and move an entire tree INSIDE used paths for quick storage space fixes, which means we're really at a best case scenario here anyway).

Look forward to @adowair's follow up..

Reply:
Yes, will fix.